README.md

```diff
@@ -97,40 +132,12 @@ $batch = array(
 islandora_job_submit_batch($batch);  // Returns TRUE
 ```
 
-Starting and Stopping Workers
------------------------------
-Workers are started and stopped programatically.  This is required so that all available jobs from hook_register_jobs() are passed to the workers on startup.  If you define a new job by adding a new entry to your module’s implementation of hook_register_jobs(), you’ll have to restart all workers for the changes to take effect.
-
-Note that both the start and stop functions take the name of a pidfile (no paths, just a name), and will respect it when starting/stopping a worker.  You just have to make sure that the apache user has permission to write files to the temporary directory defined in Drupal configuration, because this is where the module will attempt to place the pid file. 
+## Starting and Stopping Workers
 
-```php
-module_load_include('inc', 'islandora_job', 'includes/utilities');
+Workers are started and stopped via upstart. If you define a new job by adding a new entry to your module’s implementation of hook_register_jobs(), you’ll have to restart all workers for the jobs to become registered and available for the workers.
```

Comment:
A little too specific, I think, indicating upstart directly... Instead, maybe something like: "[via] system services such as those implemented in [gearman-init](https://github.com/discoverygarden/gearman-init)."